src/main/perl/Kerberos.pm

```diff
@@ -314,6 +319,18 @@ sub get_context
         return if(! defined($name));
     }
 
+    my $cred;
+
+    if ($opts{usecred}) {
+        $self->verbose("trying to get credential");
+        $cred = $self->get_cred(name => $name);
+        # Already reports error
+        return if ! $cred;
+    } else {
+        $self->verbose("No checking for credential");
```

Comment:
OK, let's leave it as verbose then. Can the messages be made consistent?
"trying to get credential" -> "Trying to get credential".
"No checking for credential" -> "Not checking for credential"